.github/workflows/build.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout 🛎️
-        uses: actions/checkout@v2
-      - uses: ljharb/actions/node/run@main
```

Comment:
This can switch to node/install and it’d work fine, i suspect - GitHub actions made some breaking changes awhile back